synthtool/gcp/templates/python_samples/README.md

```diff
@@ -0,0 +1,66 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `python -m synthtool`."
+
+## Python Sample Folder for {{ metadata['repo']['name_pretty'] }} 
+
+This directory contains samples for {{ metadata['repo']['name_pretty'] }}.
+
+## Getting Started
+-------------------------------------------------------------------------------
+
+### Prerequisites
+See the [Authentication Getting Started Guide][authentication]. Contribution guidelines, the language style guide, and license information can be found in the base parent directiory.{% if metadata['repo']['requires_billing'] %}You will need to [enable billing][enable_billing] to use Google {{ metadata['repo']['name_pretty'] }}.{% endif %}
+{% if metadata['repo']['custom_content'] is defined %}{{ metadata['repo']['custom_content']}}{% endif %}
+If this is your first time working with GCP products, you will need to set up a project. *For more information on basic setup, view the information in the parent directory's README.*
+
+### Install Dependencies
+
+1. Check to make sure you have `pip` installed and have created a `virtualenv`.
+
+1. Clone the sample folder and navigate to the sample directory you want to use.
```

Comment:
s/Clone the sample folder/Clone the branch/

Also should people be cloning the branch or making a fork of it?

Reply:
Being pretty new to the Github flow I'm not exactly how to word this. What do you suggest?

Reply:
Cloning the branch is fine. 

You can say "1. Clone the branch and navigate to the sample directory you want to use."